pages/products/index.tsx

```diff
@@ -31,8 +37,115 @@ export const Products: FC<Props> = props => (
         />
       ))}
     </ul>
-  </AppPage>
-);
+  )
+}
+
+const CreateFilterOptionsFromTaxonomies = () => Object.fromEntries(Object.entries(taxonomies.product_category.terms).map(([codename, obj]) => [codename, obj.name]));
+
+const FilterOptions = CreateFilterOptionsFromTaxonomies();
```

Comment:
I even thought that might be in a review comments :D It was because its really badly readable what I intended to do without the function :D

Reply:
How do these `() =>` few characters improve readability? If written like this `const FilterOptions = Object.fromEntries...` it would even be shorter. :)

Reply:
On top of that, when it is like this, I am not sure whether the function `CreateFilterOptionsFromTaxonomies` is reused or not. If you'd inline it into the `FilterOptions` constant there would be no function to worry about in the first place.

Reply:
btw. why are the constant and the function named with PascalCase? 😄 We use camelCase for those.